components/sections/ContactUs.jsx

```diff
@@ -1,15 +1,15 @@
 import LargeButton from "../ui/buttons/LargeButton";
-import { Heading2 } from "../ui/Typography";
+import { Heading2, Paragraph } from "../ui/Typography";
 
 const ContactUs = () => {
     return (
-        <section className="bg-primary-800 mx-0.5 rounded-8 gap-16 xl:px-0 px-5 py-5.5 md:py-16 flex flex-col items-center">
+        <section className="bg-primary-800 mx-0.5 rounded-8 gap-8 md:gap-16 xl:px-0 px-5 py-5.5 md:py-16 flex flex-col items-center">
             <div className="xl:w-3.73/5">
-                <Heading2 className="text-center text-white text-11.75">Ready to talk about your project?</Heading2>
-                <div>
-                    <p className="text-center text-white/90">Are you looking for a consultancy partnership to address your blockchain challenges? We&apos;d love to hear from you.  Let&apos;s build something</p>
-                    <p className="text-center text-white/90">amazing together.</p>
-                </div>
+                <Heading2 className="text-center">Ready to talk about your project?</Heading2>
+                <Paragraph className="mt-3 text-center text-white">
+                    Are you looking for a consultancy partnership to address your blockchain challenges?
+                    <br className="hidden md:block" />We&apos;d love to hear from you. Let&apos;s build something amazing together.
+                </Paragraph>
             </div>
             <LargeButton text="Schedule a call" styleSpan="mt-0.13" hasArrow={true} redirectLink="mailto:u2467@apeunit.com" />
```

Comment:
the button text should have a font size of  20px on the mobile screens , i suggest you use the right component of **Heading4**